### PR TITLE
2016-03-01-Django-Girls-at-Rackspace-Austin.md

### PR DESCRIPTION
**Pull request by nina-toscano**
author mini bio: Nina Toscano provides executive support to key leaders within the Rackspace Cloud Infrastructure organization. Additionally, Nina serves as a committee volunteer and leader for internal organizations including the Austin Tech Crew and the Austin chapter of the Professional Organization for Women's Empowerment at Rackspace.
author social media:none
redaction, editing by technical writer staff: accept
desired date of publishing: 3/1
willing to answer comments: decline

### CODE REVIEW

**Comment by ktbartholomew** [comment]
@nina-toscano Hey! You'll probably want to update the publish date to be a day early next week, instead of yesterday.
@catlook do your thing! :smile:

**Comment by nina-toscano** [comment]
Date has been updated. Thanks, all!

**Comment by everett-toews** [comment]
Awesome to see the PR. Go @nina-toscano go! :tada:

File: _posts/2016-02-18-Django-Girls-at-Rackspace-Austin.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+title: "Django Girls at Rackspace Austin"
+date: 2016-02-18 10:00
+comments: false
+author: Nina Toscano
+authorIsRacker: true
+published: true
+categories:
+
+---
+​​I discovered [Django Girls Austin](https://djangogirls.org/austin/) this past September when they ran a programming workshop for women in Austin. Django Girls is a non-profit organization that strives to get people excited and in love with building things. The free, one-day workshop event teaches women and girls to build their first web application using HTML, CSS, Python and Django.
```
Comment (catlook) [inline _posts/2016-02-18-Django-Girls-at-Rackspace-Austin.md]: ... get people excited and to fall in love with building  ...

File: _posts/2016-02-18-Django-Girls-at-Rackspace-Austin.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+authorIsRacker: true
+published: true
+categories:
+
+---
+​​I discovered [Django Girls Austin](https://djangogirls.org/austin/) this past September when they ran a programming workshop for women in Austin. Django Girls is a non-profit organization that strives to get people excited and in love with building things. The free, one-day workshop event teaches women and girls to build their first web application using HTML, CSS, Python and Django. 
+
+<!-- more -->
+
+Thinking I was an ideal candidate, I immediately applied for a spot as a participant in their workshop, but was unfortunately not selected. That left me wondering how I could possibly get involved with the program ... until I heard that Django Girls Austin was looking for a host for their second workshop of the year.  I knew that with the right support, I could bring this event to Rackspace Austin.
```
Comment (catlook) [inline _posts/2016-02-18-Django-Girls-at-Rackspace-Austin.md]: ... workshop but was ...

File: _posts/2016-02-18-Django-Girls-at-Rackspace-Austin.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+### All Walks Lead to Technology
+
+The program participants came from all walks of life, including a registered nurse who wants to move from the medical field into technology. Another participant is a trilingual sales consultant who wanted to add another 'language' under her belt. Then we had a mother and daughter duo who were taking the time to bond over a fun challenge.
+
+The participants and coaches coded side by side for eight hours to develop their own live web blog via Django. The tutorial led participants to become familiar with bugs and errors and understand how to read and fix them, similar to what programmers experience each day. At the end of the session, the coaches answered questions from participants about other programming languages, furthering skills to find a job as a developer, as well as 'a day in the life' queries.
```
Comment (catlook) [inline _posts/2016-02-18-Django-Girls-at-Rackspace-Austin.md]: ... errors and to understand how to ...

File: _posts/2016-02-18-Django-Girls-at-Rackspace-Austin.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+Every woman walked away energized with a new confidence in her own skills and appreciation of how programming touches her world. I have no doubt that this is the first crucial step for the participants in their exciting technology journey.
```
Comment (catlook) [inline _posts/2016-02-18-Django-Girls-at-Rackspace-Austin.md]: ...  energized, with a new ...

File: _posts/2016-02-18-Django-Girls-at-Rackspace-Austin.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+One of our core values at Rackspace is "Passion for Our Work." By bringing Django Girls to Rackspace I found a way to layer passion for my work here, with my personal passion. This combination has resulted in one highly motivated Racker! I am so proud to have been involved with this event and grateful that we have a workplace that encourages and supports programs like Django Girls.
```
Comment (catlook) [inline _posts/2016-02-18-Django-Girls-at-Rackspace-Austin.md]: ... Django Girls to Rackspace, I found ...

{Ed note:  Thanks for sharing, and I'm sorry it took me so long to respond!!}

File: _posts/2016-02-18-Django-Girls-at-Rackspace-Austin.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+I invite other Rackers who have engaged in similar community programs to share your stories. And for those of you who haven't yet, I encourage you to seek out such opportunities. I promise you will find it to be a truly rewarding experience personally, and professionally.   ​
```
Comment (catlook) [inline _posts/2016-02-18-Django-Girls-at-Rackspace-Austin.md]: ... personally and professionally.   ​

**Comment by annegentle** [comment]
Nice post @nina-toscano! :seedling:

**Comment by catlook** [comment]
@nina-toscano If you want to apply any of my suggested edits, please update your PR.  If not, it will go in with no changes applied.  Thanks again!

**Comment by nina-toscano** [comment]
@catlook - All set with your edits. Thanks, everyone!

**Comment by annegentle** [comment]
Okay if I merge on a weekend? It's a nice bit of content for the weekend, I'm thinking.

**Comment by everett-toews** [comment]
@annegentle Nina explicitly asked to have it published on 3/1.

**Comment by nina-toscano** [comment]
@annegentle - weekend works for me!

**Comment by annegentle** [comment]
Well, darn, I managed to push up a changed branch. https://github.com/rackerlabs/docs-developer-blog/pull/177

So now what?

**Comment by smashwilson** [comment]
I just set up a reminder to ship this tomorrow morning. Don't worry about the post date, I usually patch that up myself while I'm merging. :+1:

**Comment by nina-toscano** [comment]
Thank you, @smashwilson and @annegentle.

**Pull request merged**